examples/forbid_mutations/config.router.yaml

```diff
@@ -0,0 +1,9 @@
+# run a router with this configuration from the project's root directory:
+# ./router -c ./examples/forbid_mutations/config.router.yaml -s ./examples/graphql/supergraph.graphql
```

Comment:
cc @BrynCooke does this look helpful? I'm worried this would confuse users more because there is no router binary, and i would not have them compile a router to try this out

Reply:
NIT: You could add a main to the example to run the example...

Reply:
I kinda wanna show our users they don't need to compile anything in order to run this, so not quite sure